aws_serverless/template.yaml

```diff
@@ -81,6 +81,37 @@ Resources:
             Path: /api/session/create
             Method: post
 
+  GetSessionRestaurantsFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: restaurant/get/
+      Handler: app.lambda_handler
+      Runtime: python3.9
+      Environment:
```

Comment:
We can also get rid of this as well since this specific lambda is not hitting the yelp api

Reply:
Makes sense! This has been removed.